### PR TITLE
refine dump unit log

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
dump unit log is confusing

### What is changed and how it works?
refine dump unit log

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

File: dumpling/dumpling.go [hunk 1]
```diff
@@ -104,7 +104,11 @@ func (m *Dumpling) Process(ctx context.Context, pr chan pb.ProcessResult) {
 		errs = append(errs, unit.NewProcessError(err))
 	}
 
-	m.logger.Info("dump data finished", zap.Duration("cost time", time.Since(begin)))
+	if err == nil {
```
Comment (csuzhangxc) [inline dumpling/dumpling.go:107]: How about checking `errs` as we send it to `pb.ProcessResult`.

Comment (lichunzhu) [reply dumpling/dumpling.go]: addressed in https://github.com/pingcap/dm/pull/661/commits/8beb823b1ddc11c32a40345d363bf3168b2a1157

**Pull request merged**